src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java

```diff
@@ -298,4 +298,37 @@ public void testVoiceToken() {
         Assert.assertEquals("AP123", outgoing.get("application_sid"));
         Assert.assertEquals("bar", outgoingParams.get("foo"));
     }
+
```

Comment:
Given that we changed the method above to pass through the value set, I think we should also test for values besides true or blank. Can we parameterize these tests and throw in a bunch of things like true, false, "", etc?

Reply:
I don't think `""` should be possible, but I can add a test for false as well. I'm not familiar with how these tests are set up in Java, or how to parameterize them, but I'll look at some of the other tests and see if I can find an example.

Reply:
@cjcodes How's this look?